pubsub/gcppubsub/gcppubsub_test.go

```diff
@@ -50,6 +52,8 @@ type harness struct {
 	closer    func()
 	pubClient *raw.PublisherClient
 	subClient *raw.SubscriberClient
+	numTopics uint32
```

Comment:
I like to write  `// atomic` next to the declaration so readers know not to access it directly.

Reply:
Done.